quest/api/datasets.py

```diff
@@ -374,16 +375,30 @@ def open_dataset(dataset, fmt=None, **kwargs):
     m = get_metadata(dataset).get(dataset)
     file_format = m.get('file_format')
     path = m.get('file_path')
+    intake_plugin = m.get('intake_plugin')
```

Comment:
Note that the class that loads data for intake is now usually called a "driver" - there are many types of plugins. https://intake.readthedocs.io/en/latest/glossary.html